docs/sources/datasources/alertmanager.md

```diff
@@ -0,0 +1,38 @@
++++
+title = "Alertmanager"
+description = "Guide for using Alertmanager in Grafana"
+keywords = ["grafana", "prometheus", "guide"]
+aliases = ["/docs/grafana/latest/features/datasources/alertmanager"]
+weight = 1300
++++
+
+# Alertmanager data source
+
+Grafana includes built-in support for Prometheus Alertmanager. Add it as a datasource and you will be able to use the [Grafana alerting UI](https://grafana.com/docs/grafana/latest/alerting/) to manage silences, contact points and notification policies. A dropdown will appear in these pages to allow you to switch between Grafana and any configured alertmanager data sources This data source cannot return graph data and is not available in panels.
+Refer to [Add a data source]({{< relref "add-a-data-source.md" >}}) for instructions on how to add a data source to Grafana. Only users with the organization admin role can add data sources.
+
+>**Note:** New in Grafana 8.0.
```

Comment:
Remove note, as these docs are for 8.0.

Reply:
How big a deal is this? Should we mention it in What's new in Grafana 8.0? @pkolyvas, thoughts?

Reply:
Generally if we are adding a whole new topic to document something, I want it called out in the What's new. However, we have previously not done this (added docs) for alpha functionality.

Reply:
This Alertmanager helps with the new alerting. So, a mention within that topic in What's New is probably the best.

Reply:
This is part of the new alerting system, not really a stand-alone thing. I don't see an entry for new alerting at all in what's new for 8.0, and I don't think it makes sense to tackle it in the scope of this PR. Is that OK?

Reply:
> This is part of the new alerting system, not really a stand-alone thing. I don't see an entry for new alerting at all in what's new for 8.0, and I don't think it makes sense to tackle it in the scope of this PR. Is that OK?

Yes

Reply:
Ah, gotcha. @achatterjee-grafana, please make sure it is linked in the What's new doc when the alerting feature entry is updated.

Reply:
Will add this as a new data source.